README.md

```diff
@@ -147,10 +147,35 @@ following policy document
       },
       {
          "Action":[
-            "s3:*"
+            "s3:GetObject",
+            "s3:GetObjectTagging",
+            "s3:PutObjectTagging",
+            "s3:PutObjectVersionTagging",
          ],
          "Effect":"Allow",
-         "Resource":"*"
+         "Resource": ["arn:aws:s3:::<bucket-name-1>/*", "arn:aws:s3:::<bucket-name-2>/*"]
```

Comment:
```suggestion
         "Resource": [
            "arn:aws:s3:::<bucket-name-1>/*", 
            "arn:aws:s3:::<bucket-name-2>/*"
         ]
```

Reply:
Fixed in f74f0c3 along with all the others:)